tests/fsharpqa/testenv/src/NugetTestPackages/buildfscnuget.bat

```diff
@@ -1,12 +1,12 @@
 @echo off
 
-set pkgversion="1.0.0-alpha-0003"
+set pkgversion="1.0.0-alpha-0006"
 
-:: Check prerequisites
-if not '%VisualStudioVersion%' == '' goto vsversionset
-if exist "%ProgramFiles(x86)%\Microsoft Visual Studio 14.0\common7\ide\devenv.exe" set VisualStudioVersion=14.0
-if exist "%ProgramFiles%\Microsoft Visual Studio 14.0\common7\ide\devenv.exe" set VisualStudioVersion=14.0
-if exist "%VS140COMNTOOLS%" set VisualStudioVersion=14.0
+:: Check prerequisites if not '%VisualStudioVersion%' == '' goto vsversionset if
+exist "%ProgramFiles(x86)%\Microsoft Visual Studio  14.0\common7\ide\devenv.exe"
+set VisualStudioVersion=14.0  if exist  "%ProgramFiles%\Microsoft Visual  Studio
+14.0\common7\ide\devenv.exe"    set     VisualStudioVersion=14.0    if     exist
```

Comment:
about visual studio check, can we use only `VS140COMNTOOLS` and `VS120COMNTOOLS`?
this env var exists after visual studio install, should be safe right? instead of poke `%ProgramFiles(x86)%\Microsoft Visual Studio  14.0\common7\ide\` or others paths